nuclia/sdk/resource.py

```diff
@@ -63,7 +75,10 @@ def create(*args, **kwargs) -> str:
         for param in RESOURCE_ATTRIBUTES:
             if param in kwargs:
                 kw[param] = kwargs.get(param)
-        resource = ndb.ndb.create_resource(**kw)
+        try:
+            resource = ndb.ndb.create_resource(**kw)
+        except exceptions.RateLimitError as exc:
+            raise RateLimitError() from exc
```

Comment:
Should we add a debug log or something to indicate the backoff?

Reply:
I would say yes